rest_framework-stubs/fields.pyi

```diff
@@ -227,7 +217,7 @@ class SlugField(CharField):
 
 class URLField(CharField): ...
 
-class UUIDField(Field[uuid.UUID, uuid.UUID | str | int, str, Any]):
+class UUIDField(Field[uuid.UUID | None, uuid.UUID | str | int | None, str | None, Any]):
```

Comment:
I don't think this is the best way to go about this.

I suspect not all methods using these typevars _can_ deal with `None`.

And many places that use these TypeVars already explicitly specify e.g. `_VT | None`. Seems a lot safer to just update the signatures of the `to_representation()` method.

Reply:
Hrm, `IntegerField.to_representation()` actually does not allow `None`... Not sure what to make of this.

```python
    def to_representation(self, value):
        return int(value)
```